doc/changelog.rst

```diff
@@ -1,6 +1,89 @@
+-  Add `extract_location` preprocessor to extract town, city, mountains etc - anything specifiable by a location (`#1251 <https://github.com/ESMValGroup/ESMValCore/pull/1251>`__) `Javier Vegas-Regidor <https://github.com/jvegreg>`__
+-  Add ensemble statistics preprocessor and 'groupby' option for multimodel (`#673 <https://github.com/ESMValGroup/ESMValCore/pull/673>`__) `sloosvel <https://github.com/sloosvel>`__
+
+Automatic testing
+~~~~~~~~~~~~~~~~~
+
+-  Add `pandas` as dependency :panda_face:  (`#1402 <https://github.com/ESMValGroup/ESMValCore/pull/1402>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  Fixed tests for python 3.7 (`#1410 <https://github.com/ESMValGroup/ESMValCore/pull/1410>`__) `Manuel Schlund <https://github.com/schlunma>`__
+-  Remove accessing `.xml()` cube method from test (`#1419 <https://github.com/ESMValGroup/ESMValCore/pull/1419>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  Remove flag to use pip 2020 solver from Github Action pip install command on OSX (`#1357 <https://github.com/ESMValGroup/ESMValCore/pull/1357>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  Add Python=3.10 to Github Actions and switch to Python=3.10 for the Github Action that builds the PyPi package (`#1430 <https://github.com/ESMValGroup/ESMValCore/pull/1430>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  Pin `flake8<4` to keep getting relevant error traces when tests fail with FLAKE8 issues (`#1434 <https://github.com/ESMValGroup/ESMValCore/pull/1434>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  Implementing conda lock (`#1164 <https://github.com/ESMValGroup/ESMValCore/pull/1164>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  Relocate `pytest-monitor` outputted database `.pymon` so `.pymon-journal` file should not be looked for by `pytest` (`#1441 <https://github.com/ESMValGroup/ESMValCore/pull/1441>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  Switch to Mambaforge in Github Actions tests (`#1438 <https://github.com/ESMValGroup/ESMValCore/pull/1438>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  [Condalock] Update Linux condalock file (`#1437 <https://github.com/ESMValGroup/ESMValCore/pull/1437>`__, `#1445 <https://github.com/ESMValGroup/ESMValCore/pull/1445>`__, `#1447 <https://github.com/ESMValGroup/ESMValCore/pull/1447>`__, `#1451 <https://github.com/ESMValGroup/ESMValCore/pull/1451>`__, `#1458 <https://github.com/ESMValGroup/ESMValCore/pull/1458>`__, `#1461 <https://github.com/ESMValGroup/ESMValCore/pull/1461>`__, `#1463 <https://github.com/ESMValGroup/ESMValCore/pull/1463>`__, `#1464 <https://github.com/ESMValGroup/ESMValCore/pull/1464>`__, `#1465 <https://github.com/ESMValGroup/ESMValCore/pull/1465>`__, `#1466 <https://github.com/ESMValGroup/ESMValCore/pull/1466>`__) github-actions[bot]
+
```

Comment:
I wonder if we could "squash" that list of PRs into one or remove this entry from the changelog as the list of PRs will increase with time.

Reply:
I already put all of these in one line (the script created separate entries for each of them), but I also thought about removing it. It doesn't really help to include it in the changelog. What do you think, @valeriupredoi @zklaus ?

Reply:
I agree. Perhaps just include a manual entry stating the fact that the lockfile was updated?

Reply:
there are unique labels associated with these sort of PRs (condalock and automated) we can filter them out